src/backend/src/services/change-requests.services.ts

```diff
@@ -636,7 +637,7 @@ export default class ChangeRequestsService {
         otherConstraints
       } = projectProposedChanges;
 
-      await validateProposedChangesFields(projectLeadId, projectManagerId, links);
+      await validateProposedChangesFields(projectLeadId, projectManagerId);
```

Comment:
we should still be validating this for projects, just pass in an empty array of links for wpProposedChanges and make projectLeadId/projectManagerId optional